vuepress/docs/next/docs/getting-started/ent-svc.md

```diff
@@ -1,3 +1,61 @@
 ---
 sidebarDepth: 1
 ---
+
+# Auxiliary Services
+
+The following sections discuss auxiliary services and commands.
+
+## Services
+
+When the Entando Bundle CLI enables a service, a default auxiliary configuration is generated and stored as a Docker Compose YAML file in the bundle's 'svc' folder. Three services are available out of the box: `mysql`, `postgresql` and `keycloak`.
+
+- `mysql` and `postgresql` obey the same naming convention
+   - The username and password are derived from the bundle name, e.g. YOUR-BUNDLE has the username YOUR-BUNDLE-USER and the password YOUR-BUNDLE-PASSWORD
+   - A database is automatically created and given the bundle name, e.g. YOUR-BUNDLE
+
+- `keycloak` contains the Entando-related configuration
+   - The `keycloak` realm name is 'entando-dev'
+   - The console is accessible at *http://localhost:9080* with username: *admin*, password: *admin*
+   - All roles and permissions are preset, based on the Keycloak realm-config file
+   - The `keycloak.yaml`, realm-config and user config files are located in the 'keycloak' folder inside 'svc'
+
+A custom service can be added to a bundle's 'svc' folder as a Docker Compose YAML file. The file and service must have the same name, e.g. if the service filename is `mysql.yml`, then the service name must be `mysql`:
+
+``` yaml
+version: '3.7'
+
+services:
+	# since filename is mysql.yml, service name is 'mysql' as declared below:
+	mysql: 
+		image: mysql
+    ...
+```
+
+## Commands
+
+Use `ent bundle svc` to enable, disable, start, stop, restart, print the logs of, or list services. Services must be formatted as Docker Compose YAML files in the 'svc' bundle folder.
+
+To run auxiliary operations, a service must be enabled. Enabled services are listed under the 'svc' attribute of the bundle descriptor, `entando.json`. Common auxiliary service commands are detailed below.
+
+| Command | Description
+| :- | :-
+| `ent bundle svc enable <service_name>` | Adds the service(s) to the bundle’s auxiliary services
+| `ent bundle svc disable <service_name>` | Removes the service(s) from the bundle’s auxiliary services
+| `ent bundle svc start <service_name>` | Starts the enabled service(s) 
+| `ent bundle svc stop <service_name>` | Stops the enabled service(s)
+| `ent bundle svc restart <service_name>` | Restarts the enabled service(s)
+| `ent bundle svc logs <service_name>` | Prints the enabled service log(s)
+| `ent bundle svc list` | Lists all enabled services
+
+**Command details:**
+
+- `svc enable`: Adds the service(s) to the 'svc' attribute of the bundle descriptor
```

Comment:
i think this description belongs in the table above and no additional details are necessary. This one is more clear what it does. 

Same for disable. The aux services is not clear> they have to go looking for what is aux services

Reply:
?? we're talking about aux services. these are all aux services, but only if they're added to the attribute and follow the required file format / dir placement. otherwise they may be services but they're not an aux service of the bundle. did the reverse because the intro paragraph calls out the svc attribute

Reply:
was just pointing out that you end up having to read the table and the details below. I think the details section would be unnecessary if those descriptions were put in the table. 

Maybe I'm confused, but are there other services besides aux services

Reply:
? i already edited this

services is a very broad term, so absolutely